flutter_launcher_icons.yaml

```diff
@@ -1,14 +1,14 @@
 flutter_icons:
```

Comment:
Please change this to: `flutter_launcher_icons`. `flutter_icons` is deprecated, so probably why it isn't working.

Reply:
Please add a screenshot of the launcher icon when it does work with the new method.

Reply:
Been able to replicate your issue on your branch. I am looking into it then perhaps can give a suggestion.

Reply:
Okay

Reply:
I got a working hypothesis. It has to do with the flavours. Since you have already got the flavour icons generated. Rename the files as follows:

- flutter_launcher_icons-development.yaml -> `flutter_launcher_icons-development.yaml.back`
- flutter_launcher_icons-staging.yaml -> `flutter_launcher_icons-staging.yaml.back`

Then run the generator again. it should update the icons accordingly and we can merge. 

cc @NewtonMutugi

Reply:
Okay on it

Reply:
Wow! That actually worked. Thanks

Reply:
Awesome. You can push the file changes. Tested them out on this end. Then I can merge. This PR is good to go. 

Ooh and mark this conversation as resolved.

Reply:
@MillerAdulu I am thinking that application's name from fluttercon to flutterconke as part of this PR. I hope this is okay?

Reply:
Do give it a go @NewtonMutugi. Change only the labels in the following places: 
- android/app/build.gradle:84,89,94
- ios/Runner.xcodeproj/project.pbxproj:542,637,727,822,915,1007,1102,1197,1287

That should be sufficient and should only take a couple of seconds.